src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -63,10 +67,14 @@ public function action(Message $message): void
      * @param Document $webhook
      * @param Document $user
      * @param Document $project
+     * @param Database $dbForConsole
      * @return void
      */
-    private function execute(array $events, string $payload, Document $webhook, Document $user, Document $project): void
+    private function execute(array $events, string $payload, Document $webhook, Document $user, Document $project, Database $dbForConsole): void
     {
+        if ($webhook->getAttribute('enabled') === false) {
```

Comment:
```suggestion
        if ($webhook->getAttribute('enabled') !== true) {
```

nulls or undefined or 0 or empty string or other weird stuff that PHP does will also work fine if we do it like this. This will ensure only exactly `true` is allowed.